### DISCUSSION TITLE
export FilterOptionOption type

### DISCUSSION THREAD

**Discussion by tbowmo** [JedWatson/react-select#5323] [Ideas]
documentation describes FilterOptionOption as type when creating custom option filters, but it is not exported directly from react-select, and you need to use a deep link into the core react-select to get it.

I propose that we export this on a higher level, to make it easier to implement custom filter methods

**Comment by Methuselah96** [comment]
Agreed, https://github.com/JedWatson/react-select/pull/5235 was created to address this, but the author did not respond to review comments. Feel free to create a new PR that accomplishes this.
